enable/testing.py

```diff
@@ -256,7 +256,9 @@ def mouse_move(self, interactor, x, y, window=None,
                                         alt_down=alt_down,
                                         control_down=control_down,
                                         shift_down=shift_down)
-        window.get_pointer_position.return_value = (x, y)
+        if isinstance(window, _MockWindow):
```

Comment:
It would also be nice to create a test using a real Window so that users can see how it is done.